invenio_files_rest/config.py

```diff
@@ -119,3 +119,11 @@
 
 FILES_REST_FILE_TAGS_HEADER = 'X-Invenio-File-Tags'
 """Header for updating file tags."""
+
+FILES_REST_EXTRA_ENCODINGS = {
```

Comment:
I would not add this as a config, but simply hard code it in top of ``utils.py``

```python
ENCODING_MIMETYPES = {
    # ...
}
"""Mapping of encoding to MIME type."""